bank2ynab.conf

```diff
@@ -177,7 +177,7 @@ Source CSV Delimiter = ,
 Header Rows = 1
 Footer Rows = 1
 Input Columns = Date,Payee,Outflow,Inflow,Running Balance
-Date Format = %%Y-%%m-%%d
+Date Format = %Y-%m-%d
```

Comment:
The 2 last configs use `MM/DD/YYYY` - should these also be changed to `%M/%D/%Y`?

Reply:
Yep. I just did a ctrl+F for %%!